apps/src/StudioApp.js

```diff
@@ -1860,7 +1860,8 @@ StudioApp.prototype.silentlyReport = function() {
   this.report({
     app: getStore().getState().pageConstants.appType,
     level: this.config.level.id,
-    skipSuccessCallback: true
+    skipSuccessCallback: true,
+    testResult: TestResults.LEVEL_STARTED
```

Comment:
Apologies if this is a dumb question, but why do we report LEVEL_STARTED on every report instead of reporting it once when the level starts?

Reply:
"when the level starts" = on initial page load?

Let me know if this answers your question:
Initially, we wanted to report milestones more often so we can get a more accurate idea of how much time a user is spending on a level (details here if helpful: https://github.com/code-dot-org/code-dot-org/pull/36606). This milestone report is intended to allow us to measure the time spent on the level. 

However, when triggering this milestone report, we weren't recording any sort of testResult as part of the API call. Even though we now had a record that the user had started the level, that wouldn't translate back to the user as a green outline on their progress bubble. 

Reporting the progress as "LEVEL_STARTED" once when the level starts would probably be a better overall design, but the existing convention from other milestone is to record a testResult on every API call, even if that testResult will end up as a noop action.

Reply:
@jmkulwik i wonder if renaming the `LEVEL_STARTED` constant would make the more clear? right now it looks like a boolean. maybe `INITIAL_TEST_VALUE` or `LEVEL_STARTED_VALUE` or something like that?

Reply:
Makes sense now, thanks for the explanation.  I tried to think of another name but the best I could come up with was "REPORTED_PROGRESS".  Not sure if that's better than "LEVEL_STARTED" or not, up to you.